acf-recaptcha-v5.php

```diff
@@ -69,14 +73,16 @@ function render_field_settings($field) {
             'label' => __('Site Key', 'acf-recaptcha'),
             'instructions' => __('Enter your site key from Google reCAPTCHA.', 'acf-recaptcha'),
             'name' => 'site_key',
-            'required' => true,
+            'placeholder' => $this->settings['site_key'],
+            'required' => false,
         ));
 
         acf_render_field_setting($field, array(
             'label' => __('Secret Key', 'acf-recaptcha'),
             'instructions' => __('Enter your secret key from Google reCAPTCHA.', 'acf-recaptcha'),
             'name' => 'secret_key',
-            'required' => true,
+            'placeholder' => $this->settings['site_key'],
```

Comment:
I believe this should be `secret_key`.

Anyway maybe we don't want to show the key as the placeholder? Perhaps it could say "Leave blank to use default keys", what do you think?

Reply:
Good catch. Sorry - that was a case of coding before caffeine. I'm fine with changing it to the placeholder you mentioned.

I had some new ideas relating to these updates. Let me know your thoughts - I might be overthinking

1) if the keys are not present in the options table, make the key fields required. Upon saving, save them to the options table, then null them so they're not saved to the individual field.
2) If ever the user supplies the same keys as we have in the options table, null them so they're not saved to the individual field.

Reply:
Number 2 may be unnecessary (by [principle of least astonishment](https://www.wikiwand.com/en/Principle_of_least_astonishment)), but we can consider number 1.

Otherwise, it could perhaps just be a notice under `instructions`, informing users that they can add default keys in the settings page as well. Because I actually don't see ACF doing client-side validation if it's omitted - the `required` flag looks like it's just for adding a red asterisk next to the label.